### PR TITLE
Added footer & added version on config/app.php

### PR DESCRIPTION
**Pull request by saravanasai**
@saeedvaziry   I have created a footer & added version to config/app.php. As  #140 requested for this feature.

image

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -16,7 +16,15 @@
...
-
+     /*
+    |--------------------------------------------------------------------------
+    | Application Version
+    |--------------------------------------------------------------------------
+    |
+    | This value is the version of your application.
+    |
+    */
+    'version' => env('VITO_VERSION', 'v1.1.0'),
```
Comment (saeedvaziry) [inline config/app.php:27]: this means every time with every release this field needs to be updated. if you forget to update then again another release needs to be done.

Comment (saravanasai) [reply config/app.php:27]: @saeedvaziry   Yes, I thought to add a git pre-commit hook. But, I don't want to do any fancy items. Suggest to me if there is any better way to do it. Let me update this PR.

Comment (saeedvaziry) [reply config/app.php:27]: Thanks. Lets first discuss the solution. I would be in for something more automated

Comment (saravanasai) [reply config/app.php:27]: Sure. Let me check if is there any other solution & share you. we can discuss it.

Comment (saeedvaziry) [reply config/app.php:27]: Was also thinking of some solutions but looks a bit complicated.

**Pull request closed without merging**